### DISCUSSION TITLE
Is it possible to use models that require sklearn's version >0.23.2)?

### DISCUSSION THREAD

**Discussion by SergioEanX** [pycaret/pycaret#2061] [Q&A] [answered]
I need to use Cubist model compatible only  with scikit-learn>=0.24.2, is there exists a "workaround" to use it  with pycaret (tha requires version==0.23.2)?

**Comment by moezali1** [comment, marked as answer]
@SergioEanX Unfortunately right now `scikit-learn==0.23.2` is a hard dependency. This will change in the future but may take a couple of months for today for PyCaret 3.0 to release.
